mirrormanager2/xmlrpc.py

```diff
@@ -41,7 +42,11 @@
 
 @XMLRPC.register
 def checkin(pickledata):
-    config = pickle.loads(bz2.decompress(base64.urlsafe_b64decode(pickledata)))
+    uncompressed = bz2.decompress(base64.urlsafe_b64decode(pickledata))
+    try:
+        config = json.loads(uncompressed)
+    except ValueError:
+        config = pickle.loads(uncompressed)
```

Comment:
I had an idea, could we log here that a pickle got submitted/read so maybe we can monitor over the coming days how many pickle we are processing and see if that goes up or down?